src/background.js

```diff
@@ -323,7 +323,7 @@ function stripFbclid(url) {
 }
 
 async function tabUpdateListener (tabId, changeInfo, tab) {
-  await updateBrowserActionIcon(tab.url);
+  await updateBrowserActionIcon(tab);
```

Comment:
Just a side note: it would be probably enough to call `updateBrowserActionIcon` when `changeInfo.url` is set. Other than that the PR looks good to me.

Reply:
Taking the r+ whiel I can get it. 😄